src/app/components/drawer/DrawerNavigation.js

```diff
@@ -54,19 +54,18 @@ const DrawerNavigation = (props) => {
           }
           route={route}
         />
-        { drawerOpen ?
-          <Relay.RootContainer
-            Component={DrawerNavigationContainer}
-            renderFetched={
-              data => (<DrawerNavigationContainer
-                drawerOpen={drawerOpen}
-                drawerType={drawerType}
-                {...props}
-                {...data}
-              />)
-            }
-            route={route}
-          /> : null }
+        <Relay.RootContainer
```

Comment:
reviewers: this guarding for showing or hiding the drawer navigation causes the animation to stop working for showing and hiding the navigation. I am unsure of why this was put in, perhaps it was from before when we were hiding the navigation on media item pages. If this needs to go back in thats fine, just curious if its not necessary anymore